python/ray/train/tests/test_torch_utils.py

```diff
@@ -77,19 +77,26 @@ def test_multi_input_multi_dtype(self):
             )
 
     def test_tensor_column_no_memory_leak(self):
+        import gc
+
         # Test that converting a Pandas DataFrame containing an object-dtyped tensor
         # column (e.g. post-casting from extension type) doesn't leak memory. Casting
         # these tensors directly with torch.as_tensor() currently leaks memory; see
         # https://github.com/ray-project/ray/issues/30629#issuecomment-1330954556
-        col = np.empty(1000, dtype=object)
-        col[:] = [np.ones((100, 100)) for _ in range(1000)]
-        df = pd.DataFrame({"a": col})
+        def code():
+            gc.collect()
```

Comment:
Should calling `gc.collect()` before/after the code be the default behavior for `_test_some_code_for_memory_leaks`? Not sure if it makes sense to do it one-off for just this test.

Reply:
+1